client/keys.go

```diff
@@ -294,6 +305,25 @@ func (k *httpKeysAPI) Create(ctx context.Context, key, val string) (*Response, e
 	return k.Set(ctx, key, val, &SetOptions{PrevExist: PrevNoExist})
 }
 
+func (k *httpKeysAPI) CreateInOrder(ctx context.Context, dir, val string, opts *CreateInOrderOptions) (*Response, error) {
+	act := &createInOrderAction{
+		Prefix: k.prefix,
+		Dir:    dir,
+		Value:  val,
+	}
```

Comment:
Why pass in `dir` and `val` and have them in struct? Perhaps confusing to users?